### ISSUE TITLE
Indexer problem (mainly with jackett)

### ISSUE THREAD

**Issue by saitoh183** [theotherp/nzbhydra2#141]
Everything was working great for the most part until yesterday. I had to update my router firmware which caused that i had to reboot my router and modem. Now i have a new IP so my DDNS updated my A Record on zoneedit and i tested and was able to connect to my stuff via domain name. 

In hydra my indexers (all jackett) are unable to get verified since and get disabled or are Red.
My first question (which might shine some light on the problem) is 
```
2018-03-20 18:41:23.667 DEBUG --- [ool-78-thread-1] o.n.indexers.capscheck.NewznabChecker    : [Search: 72799, IP:<IP>] Calling URL https://saitohhome.com/jackett/api/v2.0/indexers/demonoid/results/torznab/api?apikey=<apikey>&t=tvsearch&tvdbid=121361
```

Should **IP** be the ip of jackett?

If so then it is not the right IP because when i ping that ip, it fails to respond. Which might point to the problem.

[nzbhydra-debuginfos-2018-03-20-14-48.zip](https://github.com/theotherp/nzbhydra2/files/1830747/nzbhydra-debuginfos-2018-03-20-14-48.zip)

**Comment by theotherp** [comment]
IP is the IP that is making the search request. Hydra gets it from the actual caller or from a header (like X-Fowarded-For, if it's set). It's for informational purposes and shouldn't affect the functionality.

According to the log Hydra was unable to connect to your jackett instance and timed out. There's much I can do to help you. I can't say why Hydra or the machine it's running on can't access your jackett.

**Comment by saitoh183** [comment]
yeah it is odd because they are on the same machine both running in docker. Maybe i will try with the docker IP but would that matter to Sonarr/Radarr that is on my seedbox?
